mappings/net/minecraft/client/gl/GlProgram.mapping

```diff
@@ -1,5 +0,0 @@
-CLASS net/minecraft/class_3679 net/minecraft/client/gl/GlProgram
```

Comment:
Is this wrong in 1.16.5, before mojang's introduction of a full shader pipeline?

Reply:
Looking at the 1.16 and 1.17 versions of this class is quite surprising, it has the same global structure with the exception of the removal of the reference counter in 1.17 and the introduction of a new internal method (which was embedded in `createFromResource` before).

So the backporting seems right here.

Looking at this case also allowed me to backport the `GlShader` interface which wasn't backported due to the fact that no mappings ever existed for that class in 1.16.